src/devices/bosch.ts

```diff
@@ -908,14 +917,90 @@ const definitions: Definition[] = [
         vendor: 'Bosch',
         description: 'Smoke alarm detector',
         fromZigbee: [fz.battery, fz.ias_smoke_alarm_1],
-        toZigbee: [],
+        toZigbee: [{
+            key: ['intruder_alarm_state', 'smoke_alarm_state', 'boschSmokeDetectorSiren', 'command'],
+            convertSet: async (entity, key, value:smokeSensorData, meta) => {
+                const dataToSend = {
+                    cluster: value?.cluster,
+                    command: value?.command,
+                    payload: value?.payload,
+                };
+                if (key === 'smoke_alarm_state' || key==='intruder_alarm_state') {
```

Comment:
What's the difference between `smoke_alarm_state` and `intruder_alarm_state`?

Reply:
Smoke alarm is triggering a different sound pattern than intruder alarm. Separated them to have 2 distinct toggles.